File: 110術科模擬試題/C.java
import java.util.*;

public class C {
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		String input;// = scanner.nextLine();
		
		int tiger=0;
		int lion=0;
		while(scanner.hasNext()){
			input = scanner.nextLine();
			//System.out.println(input);
			if(input.trim().compareTo("Tiger")==0)
				tiger++;
			else
				lion++;
		}
		if (tiger>lion)
			System.out.println("Tiger");
		else
			System.out.println("Lion");
		
	}
}

File: 110術科模擬試題/C.py
#110商業技藝競賽模擬試題
#Problem C 票選動物
#Author: Yotrew Wing
#2021/10/18
#https://github.com/yotrew/commercial_skill_competition
tiger=0
lion=0
for i in range(9):
    if input()=="Tiger":
        tiger+=1
    else:
        lion+=1
        
if tiger>lion:
    print("Tiger")
else:
    print("Lion")
